bin/codemods/src/single-tree-rendering.js

```diff
@@ -78,6 +84,38 @@ export default function transformer( file, api ) {
 	}
 
 	/**
+	 * Removes imports maintaining any comments above them
+	 *
+	 * @param {object} collection Collection containing at least one node. Comment is preserved only from first node.
+	 */
+	function removeImport( collection ) {
+		const node = collection.nodes()[ 0 ];
+
+		// Find out if import had comment above it
+		const comment = _.get( node, 'comments[0]', false );
+
+		// Remove import (and any comments with it)
+		collection.remove();
```

Comment:
Note that this wouldn't work: `node.remove()` so I'm removing the whole collection, although they will always contain only one `import` node.